.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "python.pythonPath": "/home/shikhar/flask-intro/env/bin/python3.7"
+}
```

Comment:
It is not good practice to maintain editor settings on VCS. Please clean it up

Reply:
Remove the _pycache_ also

Reply:
okay,  i deleted this .vscode